src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/Elite2ApiService.java

```diff
@@ -93,5 +94,20 @@ public String getOffenderNoFromBookingId(final Long bookingId) {
 
         return body != null ? body : Collections.emptyList();
     }
+
+    List<LocationGroup> getLocationGroups(String agencyId) {
```

Comment:
final String agencyId ?

Reply:
A few missing in LocationGroupFromEliteService.java too

Reply:
we don't have any standards around using final or not.

Reply:
I've never bothered in the past because they are effectively final anyway - but it is nicer to get some protection from the compiler.  I'll stick them in.

Reply:
If I find a project that is missing them then I would normally add them in, for fields, parameters and variables.

Reply:
Agreed use final by default